scripts/config.pl

```diff
@@ -98,6 +100,8 @@
 MBEDTLS_PKCS11_C
 MBEDTLS_NO_UDBL_DIVISION
 MBEDTLS_NO_64BIT_MULTIPLICATION
+MBEDTLS_PSA_CRYPTO_SPM
```

Comment:
Minor: while it technically doesn't hurt, this is a bit confusing as there is no such option in `config.h` (there used to be one, but it was removed while the submodule PR was reviewed and reworked).

Reply:
Thanks for spotting, I will remove it.

Reply:
Fixed in e10f191.